tests/python/gaia-ui-tests/gaiatest/brick/test_brick_verification.py

```diff
@@ -0,0 +1,19 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.homescreen.app import Homescreen
+
+
+class TestBrickVerification(GaiaTestCase):
+
+    def setUp(self):
+        # Open Firefox OS and Skip the First Time Use app
+        GaiaTestCase.setUp(self)
+        #Open the homescreen app
+        self.homescreen = Homescreen(self.marionette)
+
+    def test_verify_homescreen(self):
+        self.apps.switch_to_displayed_app()
+        self.wait_for_element_displayed(self.homescreen._homescreen_container_locator)
```

Comment:
I would wrap this call in the page class itself. Here we're using a protected property of the page class.